app/value_objects/search_resources/feed_content.rb

```diff
@@ -0,0 +1,31 @@
+module SearchResources
```

Comment:
I see that you are introducing a new "value_object" pattern to our codebase. This is something that would need our eng team's discussion and buy-in. For this moment, can you please move this content to our `app/services/search` folder? thank you.